packages/directlinespeech/__tests__/sendSpeechActivity.js

```diff
@@ -41,7 +63,7 @@ test('should echo back when saying "hello" and "world"', async () => {
 });
 
 test('should echo back "Bellevue" when saying "bellview"', async () => {
-  const { directLine, sendTextAsSpeech } = await createTestHarness();
+  const { directLine, sendTextAsSpeech } = await createTestHarness({ enableInternalHTTPSupport: false });
```

Comment:
If we set the default value for the options parameter to an empty object in the function definition, then we don't need to pass the options as a parameter to `createTestHarness` and `enableInternalHTTPSupport` will inherently be falsy. 

```suggestion
  const { directLine, sendTextAsSpeech } = await createTestHarness();
```